doc/development/api/virtual_schema_api.md

```diff
@@ -461,7 +461,19 @@ This document contains the most important metadata of the virtual schema and is
 
 ### Schema Metadata
 
-This document is usually embedded in responses from the Adapter and informs the database about all metadata of the Virtual Schema, especially the contained Virtual Tables and it's columns. The Adapter can store so called `adapterNotes` on each level (schema, table, column), to remember information which might be relevant for the Adapter in future. In the example below, the Adapter remembers the table partitioning and the data type of a column which is not directly supported in EXASOL. The Adapter has these information during pushdown and can consider the table partitioning during pushdown or can add an appropriate cast for the column.
+This document is usually embedded in responses from the Adapter and informs the database about all metadata of the Virtual Schema, especially the contained Virtual Tables and it's columns.
+
+The Adapter can store so called `adapterNotes` on each level (schema, table, column), to remember information which might be relevant for the Adapter in future. Adapter note are simple strings. You can serialize objects into those strings of course, but keep in mind that the strings are embedded inside the Virtual Schemas JSON protocol, which makes quoting of conflicting characters necessary.
+
+Some options to deal with the embedding issue:
+
+1. After serialization use Base64 encoding or
+1. Use a serialization that does not have conflicting characters like a simple CSV format or
+1. Quote conflicting characters
+
+Which variant you should choose depends on considerations like ammount of data to be transmitted, original data format and encoding overhead.
+
+In the example below, the Adapter remembers the table partitioning and the data type of a column which is not directly supported in EXASOL. The Adapter has these information during pushdown and can consider the table partitioning during pushdown or can add an appropriate cast for the column.
```

Comment:
Pre-existing typo: Replace "these information" with "this information".

Please change the example values to strings (1 example above, 3 below). I would just use the quoting approach here.

Reply:
You are right. I just learned [something new about the English language](https://jakubmarian.com/information-is-vs-informations-are-is-information-plural-or-singular-in-english/). I thought this is a case like "the police" always being plural.